dashboards.html

```diff
@@ -51,7 +51,11 @@ <h5 class="card-header">Dashboards</h5>
                     </div>
 
                     <div class="card-body">
+
+                        <a href="https://dashboards.mhss.gov.na/ptracker.html"  target="_blank"><h5 class="card-title">PTracker Dashboard and Reports</h5></a>
```

Comment:
The PTracker links must be on the PTRacker URL ptracker.mhss.gov.na/dashboard not on NDW.

Reply:
Thanks for the feedback. This has been updated